test/addons/rook-external-cluster/start

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import sys
+import time
+import shlex
+import io
+import tempfile
+import subprocess as subp
```

Comment:
We don't so such renames, they make the code harder to follow. Did you copy this code from somewhere?